### PR TITLE
Fixes radiation hopefully and metabolism

### PR DESCRIPTION
**Pull request by Bod9001**
CL: [New] plants now can produce reagents and put it into the reagent container of the food, #7841 is opened so all content is converted over to plant data instead of initial contents,
hopefully makes the radiation thread more robust,
and balances metabolic reactions, stuff like toxin was damaging the stomach causing it not being able to digest the rest of the toxin

CL: [Fix] fixed a case of radiation thread failing.

### CODE REVIEW

File: UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs [hunk 1]
```diff
@@ -56,14 +56,14 @@ public override void PossibleReaction(BodyPart sender, ReagentMix reagentMix, fl
 				{
 					foreach (var Effect in Effects)
 					{
-						sender.TakeDamage(null, Effect.EffectPerOne * 3 * LimitedreactionAmount * -OverdoseDamageMultiplier, Effect.AttackType,
+						sender.TakeDamage(null, Effect.EffectPerOne * 6 * LimitedreactionAmount * -OverdoseDamageMultiplier, Effect.AttackType,
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs]: Might be better to make this number a constant?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs]: ok

Comment (corp-0) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs]: what is that number?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs]: // This balance is about right with 1 u ingested 1 * effect it about does one damage

**Review by NoooneyDude — DISMISSED** [review]

**Review by NoooneyDude — APPROVED** [review]

**Review by ThatDan123 — APPROVED** [review]

**Pull request merged**